### DISCUSSION TITLE
求助，请问 umi@4 如何兼容到 Chrome 58 ，求相关的配置，谢谢

### DISCUSSION THREAD

**Discussion by weicong** [umijs/umi#11775] [General]
请问 umi@4 如何兼容到 Chrome 58，哪位大神可以给出相关的配置，谢谢啦

**Comment by fz6m** [comment]
当需要兼容特别旧的浏览器时，不建议使用 umi 4 ，因为 umi 是一个 react 的面向企业中后台 pc 端的框架，所以当你需要较低浏览器兼容和存在手机端场景时，则不建议使用 umi 。
